### PR TITLE
feat: updates to marker genes panel

### PR DESCRIPTION
**Pull request by joyceyan**
## Reason for Change

https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-data-portal/6132

## Changes

implements the changes detailed in the ticket.

in gene expression sidebar:
image

image

image

in cellguide:

image

image

image

image

image

note that this PR also moves some stuff into common directories so we can use things for both the marker genes table in cellguide and in GE

## Testing steps

1. go to [rdev](https://pr-6260-frontend.rdev.single-cell.czi.technology/)
2. open up any the sidebar for any cell in gene expression, and check the tooltip copy + column names
3. click into cell guide, and check the tooltip copy + column names

## Checklist 🛎️

- [x] Add product, design, and eng as reviewers for rdev review
- [x] For UI changes, add screenshots/videos, so the reviewers know what you expect them to see
- [x] For UI changes, add e2e tests to prevent regressions

### CODE REVIEW

**Comment by sidneymbell** [comment]
Nit: Could we round the effect sizes and specificity values in GE to 2 decimal places?

File: frontend/src/views/WheresMyGeneV2/common/constants.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 export const GENE_SEARCH_BAR_HEIGHT_PX = 32;
 export const HEATMAP_CONTAINER_ID = "heatmap-container-id";
 export const FMG_GENE_STRENGTH_THRESHOLD = 0.5;
+export const FMG_SPECIFICITY_THRESHOLD = 0.0;
```
Comment (atarashansky) [inline frontend/src/views/WheresMyGeneV2/common/constants.ts:4]: love the proactiveness here

File: frontend/src/views/WheresMyGeneV2/components/CellInfoSideBar/index.tsx [hunk 2]
```diff
@@ -206,32 +233,57 @@ function CellInfoSideBar({
...
+            <DivTableCell align data-testid="marker-genes-table-specificity">
+              {TABLE_HEADER_SPECIFICITY}
+              <Tooltip
+                sdsStyle="dark"
+                placement="bottom"
+                width="default"
+                className="fmg-tooltip-icon"
+                arrow
+                onOpen={() => setHoverStartTime(Date.now())}
+                onClose={handleMarkerScoreHoverEnd}
```
Comment (atarashansky) [inline frontend/src/views/WheresMyGeneV2/components/CellInfoSideBar/index.tsx:251]: I think this specificity tooltip will fire off the following (outdated) analytics event:
image
analytics need to be updated to "effect size" and a new event has to be added for specificity tooltip hover. cc @ainfeld

**Review by atarashansky — APPROVED** [review]
LGTM - analytics for specificity tooltip in wmg need to be updated, though. approving preemptively

**Pull request merged**